### PR TITLE
Update test_e2e_10_mef_eline.py

### PR DESCRIPTION
**Pull request by dgarc330**
Change the assertion method from self.assertEqual(response.status_code, 201) to self.assertEqual(response.status_code, 200) as requested in order to avoid the following assertion error:

 kytos_1    |         # enable the interfaces
kytos_1    |         api_url = KYTOS_API+'/topology/v3/interfaces/%s/enable' % (sw1if1)
kytos_1    |         response = requests.post(api_url)
kytos_1    | >       self.assertEqual(response.status_code, 201)
kytos_1    | E       AssertionError: 200 != 201
kytos_1    | 
kytos_1    | tests/test_e2e_05_topology.py:105: AssertionError

### CODE REVIEW

File: tests/test_e2e_10_mef_eline.py [hunk 1]
```diff
@@ -719,11 +719,97 @@ def evc_intra_switch_without_VLAN_tag(self):
 
     def create_many_EVC_at_once_and_verify_proper_installtion(self):
         # TODO Create many EVC at once and check if they are all working (e.g., 300 EVCs in the same file)
-        #  check if vlan-id is inside the dump-flows
-        assert True
+
+        url = "http://67.17.206.252:8181/api/kytos/mef_eline/v2/evc/"
```
Comment (hdiogenes) [inline tests/test_e2e_10_mef_eline.py:723]: I think this test should be in another PR, right? It also contains a hard-coded IP that should be modified.

**Comment by dgarc330** [comment]
I have submitted a new pull request that only reflects the changes done to the Assertion statements in order to comply with best practices
